components/opponent/commons/opponent_libraries.lua

```diff
@@ -10,8 +10,11 @@ local Lua = require('Module:Lua')
 
 local Info = Lua.import('Module:Info', {requireDevIfEnabled = true})
 
+local Opponent = Lua.import('Module:' .. (Info.opponentLibrary or 'Opponent'), {requireDevIfEnabled = true})
+local OpponentDisplay = Lua.import('Module:' .. (Info.opponentDisplayLibrary or 'OpponentDisplay'),
+	{requireDevIfEnabled = true})
+
 return {
-	Opponent = Lua.import('Module:'.. (Info.opponentLibrary or 'Opponent'), {requireDevIfEnabled = true}),
-	OpponentDisplay = Lua.import('Module:'.. (Info.opponentDisplayLibrary or 'OpponentDisplay'),
-		{requireDevIfEnabled = true})
+	Opponent = Opponent, ---@module 'opponent.commons.opponent'
+	OpponentDisplay = OpponentDisplay, ---@module 'opponent.commons.opponent_display'
```

Comment:
might this have some impact on annos for custom modules of components on wikis that have opponent custom modules?

Reply:
Theoretically, yes, but best I could do right now